src/__tests__/createSchemaForm.test.tsx

```diff
@@ -1847,4 +1858,554 @@ describe("createSchemaForm", () => {
     const inputs = screen.getAllByTestId(/dynamic-array-input/);
     expect(inputs.length).toBe(3);
   });
+  describe("FormFragment", () => {
+    it("should provide a nested renderer for use in complex components", async () => {
+      const mockOnSubmit = jest.fn();
+
+      function ComplexField({}: { complexProp1: boolean }) {
```

Comment:
is this prop supposed to be used?

Reply:
this was just to test that the props compile properly, but actually ya this is a good call to also make sure the props make it through properly